bot/bot.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+from pathlib import Path
+
+import disnake
+from dotenv import load_dotenv
+from disnake.ext import commands
+
+
+logger = logging.getLogger(__name__)
+
+path = Path(__file__)
+parent = path.parents[1]
+load_dotenv(parent.joinpath(".env"))
```

Comment:
We don't need to pass a path to load_dotenv as far as I know, it'll just find it in the project root.

Also, if we do it like this, it allows us to make dot_env a dev dep, since it's not needed in prod.
```suggestion
try:
    import dotenv
    dotenv.load_dotenv()
except ModuleNotFoundError:
    pass
```